README.md

```diff
@@ -16,8 +16,8 @@ Getting started
 
 1. From the root directory of your parent repository, copy the template to your travis configuration:
 
-        $ cp ci/travis.template.yml .travis.yml
-        $ cp ci/travis.template.env .travis.env
+        $ cp ci/travis.yml.template .travis.yml
+        $ cp ci/travis.env.template .travis.env
```

Comment:
Any reason for the rename? The nice thing about keeping the original extension is that syntax highlighting continues to work.

No big deal, but if you go through with it, check the links and references elsewhere on this page. Line 22 still points to the old name.

Reply:
My bad, didn't realize it was already named that way. Still, check line 22.